optuna/artifacts/exceptions.py

```diff
@@ -5,8 +5,8 @@ class ArtifactNotFound(OptunaError):
     """Exception raised when an artifact is not found.
 
     It is typically raised while calling
-    :meth:`~optuna_dashboard.artifact.protocol.ArtifactBackend.open` or
-    :meth:`~optuna_dashboard.artifact.protocol.ArtifactBackend.remove` methods.
+    :meth:`~optuna.artifact.protocol.ArtifactBackend.open_reader` or
```

Comment:
Hmm, I'm not sure but ArtifactStore is in a private submodule.
```suggestion
    :meth:`~optuna.artifacts._protocol.ArtifactStore.open_reader` or
```